Frontend/Frontend.Client/Components/MessageUi.razor

```diff
@@ -1,9 +1,11 @@
+@namespace Frontend.Client.Components
```

Comment:
Can we please rename this file ti MessageComponent?

Reply:
ok